java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/text/ArrowFlightJdbcVarCharVectorAccessorTest.java

```diff
@@ -584,17 +584,21 @@ public void testShouldGetTimestampReturnValidDateWithCalendar() throws Exception
 
   private void assertGetBoolean(Text value, boolean expectedResult) throws SQLException {
     when(getter.get(0)).thenReturn(value);
-
     boolean result = accessor.getBoolean();
     collector.checkThat(result, equalTo(expectedResult));
   }
 
-  @Test
-  public void testShouldGetBooleanReturnTrueForNonEmpty() throws Exception {
-    assertGetBoolean(new Text("anything"), true);
+  private void assertGetBooleanForSQLException(Text value) {
+    when(getter.get(0)).thenReturn(value);
+    collector.checkThrows(SQLException.class, () -> accessor.getBoolean());
   }
 
   @Test
+  public void testShouldGetBooleanReturnTrueForNonEmpty() {
+    assertGetBooleanForSQLException(new Text("anything"));
+  }
+
+  @Test(expected = SQLException.class)
```

Comment:
Use `assertGetBooleanForSQLException` for this test too

Reply:
fixed